src/haz3ltest/haz3ltest.re

```diff
@@ -0,0 +1,10 @@
+open Alcotest;
+open Js_of_ocaml;
+open Test_Elaboration;
+
+let main = (_s: string) => {
+  run("Dynamics", [("Elaboration", elaboration_tests)]);
+};
+
+Js.Unsafe.js_expr("require('process')");
+Js.Unsafe.js_expr("process.argv[2]") |> Js.to_string |> main;
```

Comment:
what is this code doing? the argument to main doesn't seem to be used so is this necessary?